src/components/Burger.vue

```diff
@@ -0,0 +1,7 @@
+<template></template>
+
+<script>
+export default {};
+</script>
+
+<style></style>
```

Comment:
This was a file for testing, right? if it is, please delete it.

Reply:
Thanks, I had forgotten about this